### DISCUSSION TITLE
Welcome to Algo_Guide Discussions!

### DISCUSSION THREAD

**Discussion by kodeflap** [kodeflap/Algo_Guide#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself.

**Comment by Montekkundan** [comment]
Hello my name is Montek, Im 18, currently a student learning about Data Science ML and cybersecurity. What I do in this community? I contribute to it!

**Reply by kodeflap** [reply]
Welcome to the community.
